ultimate_cron.install

```diff
@@ -209,7 +207,7 @@ function ultimate_cron_requirements($phase) {
         );
         $requirements['description'][] = t(
           'Some jobs are behind their schedule. Please check if !link is running properly.', array(
-          '!link' => l(t('cron'), 'admin/config/system/cron')
+          '!link' =>  'admin/config/system/cron',
```

Comment:
Or Drupal::l(). But it's quite common for transltable strings to include the <a> tags and just inject the url.